velox/exec/tests/HashJoinTest.cpp

```diff
@@ -5856,4 +5856,56 @@ DEBUG_ONLY_TEST_F(HashJoinTest, hashProbeAbortDuringInputProcessing) {
     waitForAllTasksToBeDeleted();
   }
 }
+
+TEST_F(HashJoinTest, leftJoinWithMissAtEndOfBatch) {
```

Comment:
Can you also create a test case that there is a leftover from previous batch but no overflow in the next one. We expect a data correctness failure instead of crash without this fix? Thanks!